tests/unit/amo/components/TestSearchForm.js

```diff
@@ -271,6 +271,13 @@ describe(__filename, () => {
 
       wrapper.find('input').simulate('change', createFakeChangeEvent(undefined));
       expect(wrapper.state('searchValue')).toEqual('');
+
+      // Tests for allowing only 100 characters as input
+      wrapper.find('input').simulate('change', createFakeChangeEvent('asdfghjklpasdfghjklpasdfghjklpasdfghjklpasdfghjklpasdfghjklpasdfghjklpasdfghjklpasdfghjklp98765432111'));
```

Comment:
haha maybe we could use [`String.repeat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat):

```
'c'.repeat(100)
```